web/src/components/core/Installation.jsx

```diff
@@ -19,19 +19,11 @@
  * find current contact information at www.suse.com.
  */
 
-import React, { useEffect, useState } from "react";
-import { useInstallerClient } from "~/context/installer";
+import React from "react";
 import { InstallationProgress, InstallationFinished } from "~/components/core";
 import { IDLE } from "~/client/status";
 
-function Installation() {
-  const client = useInstallerClient();
-  const [status, setStatus] = useState(undefined);
-
-  useEffect(() =>
-    client.manager.onStatusChange(setStatus), [client.manager]
-  );
-
+function Installation({ status }) {
```

Comment:
Who is passing the status to this component?

Reply:
oh, I forgot to commit that change. Done.